### PR TITLE
feat: use voyager subscriptions package

### PR DESCRIPTION
**Pull request by darahayes**
### Description

Related PRs:

https://github.com/aerogear/aerogear-js-sdk/pull/346
https://github.com/aerogear/voyager-server/pull/164

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by wtrocki** [comment]
ping @StephenCoady There will be small clash with the code here and in #171

**Comment by StephenCoady** [comment]
@wtrocki thats fine, we can merge this PR first and I can break the subscriptions back into individual ones.

File: server/src/subscriptions.js [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-    }, {
-      server: httpServer,
-      path: '/graphql'
-    });
-}
 
 module.exports = {
-    pubSub: new PubSub(),
-    subscriptionServer
+  pubSub: new PubSub()
```
Comment (StephenCoady) [inline server/src/subscriptions.js]: I think we should just move this to the subscriptions schema file in tasks if its the only thing this file is being used for. thats the only place it is used so a one liner there makes sense.

Comment (darahayes) [reply server/src/subscriptions.js]: +1 on that point.

**Comment by darahayes** [comment]
@wtrocki @StephenCoady So there are 3 PRs going on here.

1. Tiny PR to the js sdk
2. Pretty big PR to create a new subscriptions package
3. This PR here which is actually pretty small

I suggest we approach it in the following way.

* Get the PR to the JS SDK merged first.
* Cut a new release of the JS-SDK if possible. (Possibly a pre release?) Question to @wtrocki and @StephenCoady is there any outstanding work on the master branch that should not be released? We could cut a release candidate in any case.
* Work to get the subscriptions package PR merged.
* Create a prerelease of the server packages
* Now we can simplify testing of the end to end functionality in the showcase app + server by using the release candidates.

What do you guys think?

**Comment by darahayes** [comment]
Hey folks, closing this PR in favour of #184

**Pull request closed without merging**